ui/app/styles/docs.scss

```diff
@@ -0,0 +1,101 @@
+#docfy-content {
+  font-size: 16px;
+  font-family: var(--token-typography-display-300-font-family);
+  margin-left: 1.5rem;
+  margin-right: 2rem;
+  margin-top: 1.5rem;
+  max-width: 90%;
```

Comment:
nit: can we make the margin-top match the headers in the original pages in the app? 😬 I played around with `margin-top: 6rem` and removing margin-left and margin-right and changing the width to `100%`. You'll also need to add a `container` class to the where `docfy-content` id is placed https://github.com/hashicorp/vault/pull/24092/files#diff-238228438f75d4420ccfab19fd10d2bf9b14049e2ecdbbad915aa44a14d5a515R73